spec/requests/v0.1/actions_spec.rb

```diff
@@ -22,7 +23,8 @@
     context 'when the record exists' do
       it 'returns the action' do
         expect(json).not_to be_empty
-        expect(json['id']).to eq(id)
+        expect(json['id']).to eq(id.to_s)
+        expect(json['created_at']).to eq(created_at.iso8601.to_s)
```

Comment:
Minor but, I would drop the extract `let` blocks for the specific properties and only do the lookup once in the test with `action = actions.first` and then `expect` would test each property:
```
action = actions.first

expect(json['id']).to eq(action.id.to_s)
expect(json['created_at']).to eq(action.created_at.iso8601)
```

Also, `iso8601` returns a string so you can drop the `to_s`.

Reply:
@gmcculloug Changed accordingly.